R-packages/covidcast/R/covidcast.R

```diff
@@ -163,6 +163,9 @@ MAX_RESULTS <- 3649
 #'   with `time_value` of June 3 will only be included in the results if its
 #'   data was issued or updated on June 6. If `NULL`, the default, return the
 #'   most recently issued data regardless of its lag.
+#' @param time_type The temporal resolution to request this data. Most signals
```

Comment:
What happens if a user requests data at a different resolution?  For daily signals is it aggregated?  For weekly signals is it split?

Reply:
Whatever the answer is, would be good to add tests to cover those cases.

Reply:
It depends on if that time_type is in the API. The API has to explicitly offer every time_type; if it doesn't, the client just reports the error for a missing or non-existent signal. There's a separate issue here, which is that the API docs will need to expand to mention the time_type available for every signal.

Reply:
I opened https://github.com/cmu-delphi/delphi-epidata/pull/335 to make sure this is clearly documented for each source